packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/document/services/DocumentSanitizedJsonContentProvider.java

```diff
@@ -46,6 +53,8 @@
 @Service
 public class DocumentSanitizedJsonContentProvider implements IDocumentSanitizedJsonContentProvider {
 
+    private final FadeButtonDescriptionProvider fadeButtonDescriptionProvider;
```

Comment:
Huh?

Reply:
Fixed. Looks like an automatic imported added at some point by Eclipse that I did not see.

Reply:
Or some issue with git applying the patch I exported from another clone.
I don't even know about this class so it was not voluntary.